test/deprecated/iochannel.chpl

```diff
@@ -0,0 +1,13 @@
+use IO;
```

Comment:
There's an IO sub-directory in test/deprecated, so it might make sense to move it there (but I'd totally understand you being tired of this change so it's okay to not move it since it will hopefully get removed soon)